src/commands/misc/man.rs

```diff
@@ -0,0 +1,137 @@
+use crate::{Context, Error};
+use poise::serenity_prelude::{
+    ComponentInteractionCollector, CreateActionRow, CreateButton, CreateInteractionResponse,
+    CreateInteractionResponseMessage,
+};
+use poise::CreateReply;
+use std::process::Command;
+use std::time::Duration;
+use tracing::{debug, warn};
+
+const MAN_VIEW_TIMEOUT: Duration = Duration::from_secs(10 * 60);
+const MSG_MAX_LEN: usize = 1950;
+
+fn remove_first_and_last_line(input: String) -> String {
+    let lines: Vec<&str> = input.lines().collect();
+    if lines.len() <= 2 {
+        return String::new();
+    }
+    lines[1..lines.len() - 1].join("\n")
```

Comment:
I'd advise returning the lines here instead of joining them, we need them later.